src/Contract/Cbor.purs

```diff
@@ -0,0 +1,18 @@
+module Contract.Cbor
+  ( module ByteArray
+  , module CborBytes
+  , hexToCbor
+  ) where
+
+import Contract.Prelude
+
+import Ctl.Internal.Types.ByteArray (hexToByteArray) as ByteArray
```

Comment:
`hexToByteArray` is exported from Contract.Prim.ByteArray, no need to re-export

Reply:
Fixed